SignNow.Net.Test/AcceptanceTests/GetTokenByAuthCodeAsyncTest.cs

```diff
@@ -0,0 +1,157 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SignNow.Net;
+using SignNow.Net.Model;
+using SignNow.Net.Test;
+using SignNow.Net.Test.Context;
+using System;
+using System.Threading.Tasks;
+
+namespace AcceptanceTests
+{
+    [TestClass]
+    public class GetTokenByAuthCodeAsyncTest : ApiTestBase
```

Comment:
Class should be named after service (OAuth2ServiceTest). Every test method should have self-descriptive name that describes what is actually checked in the test